manifests/h/HexRays/IDA/Free/7.0/HexRays.IDA.Free.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: HexRays.IDA.Free
+PackageVersion: "7.0"
+PackageLocale: en-US
+Publisher: Hex-Rays SA
+PublisherUrl: https://hex-rays.com/
+PublisherSupportUrl: https://hex-rays.com/products/ida/support/
+PrivacyUrl: https://hex-rays.com/privacy_policy/
+Author: Hex-Rays SA
+PackageName: IDA Freeware and Hex-Rays Decompilers (x64)
```

Hmm, does this cause a duplicate PackageName issue?
```suggestion
PackageName: IDA Freeware
```